tahrir_api/scripts/populatedb.py

```diff
@@ -0,0 +1,217 @@
+
+import datetime
+import os
+import random
+import sys
+import transaction
+
+from sqlalchemy import engine_from_config
+from sqlalchemy.orm import scoped_session, sessionmaker
+from paste.deploy import appconfig
+from zope.sqlalchemy import ZopeTransactionExtension
+
+from ..model import (
+    Issuer,
+    Badge,
+    Person,
+    Team,
+    Milestone,
+    Series,
+    Assertion,
+    DeclarativeBase,
+)
+from ..utils import convert_name_to_id
+
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension(),
+                                        expire_on_commit=False))
+
+
+def usage(argv):
```

Comment:
You might consider using ```click``` to make this CLI tool - it generates a really nice usage help for you (and a ```--help``` too!)